version-embed.go

```diff
@@ -6,8 +6,12 @@ package tailscaleroot
 
 import _ "embed"
 
+// VersionDotTxt is the contents of VERSION.txt. Do not use this
+// directly as the binary's version, use the version package
+// instead.
```

Comment:
If you're going to say no, say a bit about why.

```
// VersionDotTxt is the contents of VERSION.txt. Despite the tempting filename,
// this does not necessarily contain the version number of the build, which depends
// on the branch type and how it was built. To get version information, use the
// version package instead.
```

Reply:
Done.